src/Common/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -11,5 +11,7 @@ internal static class HttpHandlerDefaults
         public const int DefaultMaxAutomaticRedirections = 50;
         public const DecompressionMethods DefaultAutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
         public const bool DefaultAutomaticRedirection = true;
+        public const bool DefaultUseCookies = true;
```

Comment:
Yes. It should be true. 
With the default value set to true, we may have another issue #3016 I haven't read the code completely